Apps/Examples/Examples/All Examples/SwitchStylesExample.swift

```diff
@@ -22,48 +68,31 @@ public class SwitchStylesExample: UIViewController, ExampleProtocol {
 
```

Comment:
> `        let mapInitOptions = MapInitOptions(cameraOptions: cameraOptions, styleURI: .satelliteStreets)`

As we are using `var style: Style` for style operations, we shouldn't use other URI types anymore. Every style assignment should go through the `style` variable.